tests/format/js/nullish-coalescing/nullish_coalesing_operator.js

```diff
@@ -20,3 +20,5 @@ foo || (baz ?? baz);
 
 (foo && baz) ?? baz;
 foo && (baz ?? baz);
+
+baz ?? baz ? a : b;
```

Comment:
Since we are not checking ternary properties, please add test for every possible cases.

```suggestion
foo ?? bar ? a : b;
a ? foo ?? bar : b;
a ? b : foo ?? bar;
```

Reply:
thank you for the suggestion! I've updated the snapshot test with these changes